Packs/Flashpoint/Integrations/Ignite/Ignite.py

```diff
@@ -651,13 +651,13 @@ def prepare_incidents_from_compromised_credentials_data(response: dict, next_run
     severity = demisto.params().get('severity', DEFAULT_SEVERITY)
 
     for hit in hits:
-        name = hit.get('_source', {}).get('email')
-        if not name:
-            name = hit.get('_source', {}).get('fpid', 'Compromised Credential Alert')
+        hit_source = hit.get('_source', {})
+        name = hit_source.get('username', hit_source.get(
+            'email', hit_source.get('fpid', 'Compromised Credential Alert')))
```

Comment:
It might be worth adding in a log line to specify which of the fields was selected for the name. Purely for debugging purposes.